src/components/SideBar/SideBarStyled.js

```diff
@@ -1,61 +1,88 @@
 import styled from 'styled-components';
+import { LuLogOut, LuMessagesSquare } from 'react-icons/lu';
+import { TbUser, TbUsers } from 'react-icons/tb';
 
-export const Wrapper = styled.div`
+export const SideBarStyled = styled.div`
   display: flex;
   flex-direction: column;
-  justify-content: space-between;
   align-items: center;
-  min-width: 120px;
   height: 100vh;
-  border-right: 1px solid #c8c8c8;
-  /* background: #E9F0FB; */
   background: var(--color-blue-1);
 `;
 
-export const ProfileBox = styled.div`
+export const ButtonsFrame = styled.div`
   display: flex;
   flex-direction: column;
   justify-content: space-between;
-  align-items: center;
-  width: 56px;
-  height: 76px;
-  margin-top: 48px;
-  cursor: pointer;
+  height: 100%;
+  margin: 48px 0;
+  margin: 48px 0;
+`;
+
+export const Text = styled.p`
+  line-height: 20px;
+  letter-spacing: -0.011em;
+  color: var(--color-grey-9);
+`;
+
+export const UserIcon = styled(TbUser)`
+  width: 24px;
```

Comment:
Seems like all icons below uses the same styles. We should make a bit clear and move these styles to a variable and use this variable for the rest of icons.